src/Symfony/Component/Validator/Tests/Mapping/ClassMetadataTest.php

```diff
@@ -47,11 +50,25 @@ public function testAddConstraintDoesNotAcceptValid()
 
     public function testAddConstraintRequiresClassConstraints()
     {
-        $this->expectException('Symfony\Component\Validator\Exception\ConstraintDefinitionException');
+        $this->expectException(ConstraintDefinitionException::class);
```

Comment:
Needless conflict with lower branches, could be reverted?

Reply:
Indeed